common/model/opening-hours.js

```diff
@@ -10,22 +10,37 @@ export type OpeningHoursDay = {|
   note?: string
 |};
 
-type ExceptionalOpeningHoursDay = {|
+export type ExceptionalOpeningHoursDay = {|
   ...OpeningHoursDay,
-  overrideDate: Date
+  overrideDate: Date,
+|}
+
+export type SpecialOpeningHours = {|
+  opens: string,
+  closes: string,
+  validFrom: Date,
+  validThrough: Date
 |}
 
 export type OpeningHours = {|
   regular: OpeningHoursDay[],
-  exceptional?: ExceptionalOpeningHoursDay[]
+  exceptional?: ?ExceptionalOpeningHoursDay[]
```

Comment:
We should be trying to avoid double maybe types.
Either it's optional, or we can't pass it a null value and don't have to pass it.

They say slightly different things about the type.